data.json

```diff
@@ -89,6 +89,15 @@
     "dateAdded": "2018-02-18T07:48:58.000Z",
     "tags": ["clothing", "hacktoberfest", "expired", "stickers"]
   },
+  {
+    "name": "Digital Ocean",
+    "difficulty": "easy",
+    "description": "Complete this survey to receive a limited edition open source 'Sammy Shark' sticker!",
+    "reference": "https://digitalocean.getfeedback.com/r/p44ADlqG/",
+    "image": "https://pbs.twimg.com/media/EExmj5fXoAAnF9B?format=jpg&name=medium",
+    "dateAdded": "2019-10-01T11:00:00.000Z",
```

Comment:
We use the date of issue creation as dateAdded.

```suggestion
    "dateAdded": "2019-08-30T10:39:37.000Z",
```

Reply:
It's not allowing me to commit suggestion. Weird!
`You do not have permission to push to this repository.`

Reply:
![image](https://user-images.githubusercontent.com/8191198/65974976-2cc2a500-e46e-11e9-865a-0f1aff3362cb.png)

I guess @jnnck unchecked this box